src/context.ts

```diff
@@ -370,35 +428,39 @@ export default class Context {
 				if (experiment.data.audienceStrict && assignment.audienceMismatch !== false) {
 					assignment.variant = 0;
 				} else if (experiment.data.fullOnVariant === 0) {
-					if (unitType in this._units) {
-						const unit = this._unitHash(unitType);
-						const assigner =
-							unitType in this._assigners
-								? this._assigners[unitType]
-								: (this._assigners[unitType] = new VariantAssigner(unit));
-						const eligible =
-							assigner.assign(
-								experiment.data.trafficSplit,
-								experiment.data.trafficSeedHi,
-								experiment.data.trafficSeedLo
-							) === 1;
-
-						assignment.assigned = true;
-						assignment.eligible = eligible;
-
-						if (eligible) {
-							if (hasCustom) {
-								assignment.variant = this._cassignments[experimentName];
-								assignment.custom = true;
-							} else {
-								assignment.variant = assigner.assign(
-									experiment.data.split,
-									experiment.data.seedHi,
-									experiment.data.seedLo
-								);
+					if (unitType !== null) {
```

Comment:
Can we keep  the same indentation level here and just add the !== null check to the same condition? This logic is very precise and I am afraid I may miss something otherwise.

Reply:
I believe this is okay with `?w=1` at the end of the Github URL?